src/Text/tests.jsx

```diff
@@ -26,3 +26,46 @@ describe( 'Text', () =>
         expect( Wrapper.find( '.text__default' ) ).to.have.length( 1 );
     } );
 } );
+
+
+describe( 'TextDriver', () =>
+{
+    let wrapper;
+    let driver;
+
+    beforeEach( () =>
+    {
+        wrapper = mount( <Text /> );
+        driver  = wrapper.driver();
+    } );
+
+    describe( 'getContent', () =>
+    {
+        it( 'should return the content set by text prop', () =>
+        {
+            const text = 'the quick brown fox jumps over the lazy dog';
+            wrapper.setProps( { text } );
+            expect( driver.getContent() ).to.equal( text );
+        } );
+
+        it( 'should return the content set by children prop', () =>
+        {
+            const text = 'the quick brown fox jumps over the lazy dog';
+            const children = <div>{ text }</div>;
+
+            wrapper.setProps( { children } );
```

Comment:
Is this completely equivalent to mounting straight on:
```
<Text>
     <div>I can be original</div>
</Text>
```
?

Cause this what will happen most likely here and there.

Reply:
yep it’s the same